pyodide_build/common.py

```diff
@@ -383,6 +383,19 @@ def _get_sha256_checksum(archive: Path) -> str:
     return h.hexdigest()
 
 
+def _format_dep_chain(dep_chain: Sequence[str]) -> str:
+    return " -> ".join(dep.partition(";")[0].strip() for dep in dep_chain)
+
+
+def _format_missing_dependencies(missing) -> str:
```

Comment:
Could you please add a type for the parameter?

Reply:
Done, thanks! Addressed in 206a28e68c8b731827e7876048beb90f16179607

Reply:
Done, thanks! Addressed in 206a28e68c8b731827e7876048beb90f16179607.